cli_tools_tests/e2e/gce_image_import_export/test_suites/import/import_tests.go

```diff
@@ -29,14 +29,12 @@ import (
 	"time"
 
 	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/path"
-	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/assert"
-	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_e2e_test/common/compute"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_tests/e2e"
+	"github.com/GoogleCloudPlatform/compute-image-tools/common/gcp"
+	logging2 "github.com/GoogleCloudPlatform/compute-image-tools/common/logging"
```

Comment:
Just logging? Or commonlogging?

Reply:
thanks; changed to 'logging' (poor automatic refactoring on my part)